src/views/open-api-docs-view/server.ts

```diff
@@ -101,7 +101,7 @@ export async function getStaticProps({
 			: await fetchGithubFile(sourceFile)
 	const schemaData = await parseAndValidateOpenApiSchema(schemaFileString)
 	const { title } = schemaData.info
-	const operationProps = getOperationProps(schemaData)
+	const operationProps = await getOperationProps(schemaData)
```

Comment:
Had to update this file and a few other files to get the `async` flow going for markdown processing.